docs/tutorials/build.mdx

```diff
@@ -60,17 +81,43 @@ This command will:
 You should see output similar to:
 ```
 Building expr from pipeline.py
-Written 'expr' to artifacts/2a3b46d8d3d0
+Written 'expr' to artifacts/3350466c8fcd
 ```
 
-## Use Cases
+<Warning>
+  Currently, we do not support the building of expressions in-memory tables
```

Comment:
I think it's more like we are yet to implement support for in-memory tables for serialization since we are still working out the best way to support tables/dataframes on disk. 

Perhaps:

> Currently, the `build` command does not support in-memory tables as it's unclear what the best approach for serializing/deserializing DataFrames for build artifacts would be.